frontend/common/src/constants/localizedConstants.tsx

```diff
@@ -1017,6 +1018,34 @@ export const getGovEmployeeType = (
     `Invalid Government of Employee Type '${govEmployeeTypeId}'`,
   );
 
+export const simpleGovEmployeeType = defineMessages({
```

Comment:
If you were just trying to get rid of the formatting I would think there would be a better way of doing it.  But you are getting much shorter description, too, which makes this worth it I think.

Reply:
Yeah, if you have some suggestion I can implement them. I though it would be easier and nicer to have a nice short description in the CSV.